plexpy/config.py

```diff
@@ -547,6 +547,7 @@ def bool_int(value):
     'TELEGRAM_ON_PMSUPDATE': (int, 'Telegram', 0),
     'TELEGRAM_ON_CONCURRENT': (int, 'Telegram', 0),
     'TELEGRAM_ON_NEWDEVICE': (int, 'Telegram', 0),
+    'THEMOVIEDB_APIKEY': (str, 'General', '2da93fe08a97c9a6d4205ff5b63ccf83'),
```

Comment:
Yes. The TMDB API is rate limited by IP address, not by the key, so sharing it should be okay.

https://www.themoviedb.org/talk/512403ec760ee37257037fdb